yokscr.vcxproj

```diff
@@ -85,6 +85,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <TargetExt>.scr</TargetExt>
     <TargetName>yok</TargetName>
+    <RunCodeAnalysis>false</RunCodeAnalysis>

```

Comment:
This doesn't do anything, false is the default here, this is just what I get for playing with the project settings too much.